4.21/ExampleProject/Plugins/PlayFab/Source/PlayFabCommon/Public/PlayFabCommonSettings.h

```diff
@@ -41,7 +41,7 @@ namespace PlayFabCommon
         {
             if (serverURL.Len() == 0)
             {
-                serverURL = TEXT("https://") + (!verticalName.IsEmpty() ? verticalName : titleId) + productionEnvironmentURL;
+                serverURL = TEXT("https://") + titleId + (!verticalName.IsEmpty() ? "." + verticalName : "") + productionEnvironmentURL;
```

Comment:
>!verticalName.IsEmpty() [](start = 58, length = 23)

I would invert this expression to avoid the !; (verticalName.IsEmpty()? "" : "." + verticalName)

Reply:
Done